pantheon-bundle/src/main/resources/SLING-INF/content/docs/assemblies/assembly-configuring-repository.adoc

```diff
@@ -0,0 +1,20 @@
+ifdef::context[:parent-context: {context}]
+
+include::../attributes.adoc[]
+
+[id="configuring_repository_{context}"]
+= Configuring your repository
+
+[role="_abstract"]
+Before you can add {ContentTerm} to {ProductName}, you must complete the following steps:
```

Comment:
Hey @stoobie - looking good! The only change I'd suggest is to the abstract here. Only this first sentence will be considered the abstract, so it should probably be a self-contained, complete sentence. Rather than referring to the next steps using XREFs, how about calling out the general steps. I.e. '...you must configure your documentation repository as a... and create a product and...'

Reply:
I didn't notice the abstract, good point.

For future reference, is this a general rule for abstracts?